docs/application/dotnet/tutorials/sec-privileges.md

```diff
@@ -78,6 +79,8 @@ when using security-sensitive API modules in .net applications:
 | `http://tizen.org/privilege/contact.read` | public | Contacts | The application can read user's profile, contacts, and contact history. Contact history can include social network activity. |
 | `http://tizen.org/privilege/contact.write` | public | Contacts | The application can create, update, and delete user's profile, contacts, and any contact history that is related to this application. Contact history can include social network activity. |
 | `http://tizen.org/privilege/content.write` | public |  | The application can change media information. This information can be used by other applications. |
+| `http://tizen.org/privilege/d2d.datasharing` | public | | The application can change the settings for services provided by the display server, such as the quick panel and softkey bar. |
```

Comment:
"The application can change the settings for services provided by the display server, such as the quick panel and softkey bar."

Is this sentence really for "http://tizen.org/privilege/d2d.datasharing"?

Reply:
Thank you. Fixed.